src/pages/solve/MaxCut.tsx

```diff
@@ -18,29 +27,45 @@ const MaxCut: NextPage = () => {
   }
 
   return (
-    <TextInputMask
-      title="MaxCut Solver"
-      description="For a given undirected, weighted graph, this algorithm finds a cut that is a maximum in some way or another."
-      textPlaceholder="Enter your graph in GML format"
-      onTextChanged={change}
-      body={
-        <Container>
-          <Center>
-            <GraphArea
-              graphData={graphData}
-              graphHeight={500}
-              graphWidth={500}
-            />
-          </Center>
+    <Layout>
+      <Heading as="h1">MaxCut Solver</Heading>
+      <Text color="text" align="justify">
+        For a given undirected, weighted graph, this algorithm finds a partition
```

Comment:
```suggestion
        For a given (un)directed, (weighted) graph, this algorithm finds a partition
```
We could almost cut directed and weighted at that point as we support both variants each.

Reply:
I'll remove the `undirected` adjective completely